ClientTests/TabEventHandlerTests.swift

```diff
@@ -38,43 +38,28 @@ class TabEventHandlerTests: XCTestCase {
         XCTAssertTrue(handler.isFocused!)
     }
 
-    func testOnlyRegisteredForEvents() {
-        let tab = Tab(configuration: WKWebViewConfiguration())
-        let handler = DummyHandler()
-        handler.doUnregister()
-
-        let tabObservers = handler.registerFor(.didGainFocus)
-
-        XCTAssertNil(handler.isFocused)
-
-        TabEvent.post(.didGainFocus, for: tab)
-        XCTAssertTrue(handler.isFocused!)
-
-        TabEvent.post(.didLoseFocus, for: tab)
-        XCTAssertTrue(handler.isFocused!)
-
-        handler.unregister(tabObservers)
-    }
+//    func testOnlyRegisteredForEvents() {
+//        let tab = Tab(configuration: WKWebViewConfiguration())
+//        let handler = DummyHandler()
+//        handler.doUnregister()
+//        let tabObservers = handler.registerFor(tabEvent: .didGainFocus)
+//        XCTAssertNil(handler.isFocused)
+//        TabEvent.post(.didGainFocus, for: tab)
+//        XCTAssertTrue(handler.isFocused!)
+//        TabEvent.post(.didLoseFocus, for: tab)
+//        XCTAssertTrue(handler.isFocused!)
+//        handler.unregister(tabObservers)
+//    }
```

Comment:
Will remove this. I was debating keeping it, but I think it is such an edge case test that it isn't needed.